src/terrain/services/teams.clj

```diff
@@ -23,7 +23,10 @@
   (ipg/get-team-members user name))
 
 (defn add-team-members [{user :shortUsername} name {:keys [members]}]
-  (ipg/add-team-members user name members))
+  (let [response (ipg/add-team-members user name members)]
```

Comment:
This is vanishingly minor and iplant-groups may not support it, but I'd imagine it's possible for not every member to successfully be added (e.g., if someone's already a member). Would it be possible to inspect the response and only send notifications for those members who were actually newly added?

Reply:
`iplant-groups` currently returns an error if any member addition fails. I tested this, and the one correct member actually wasn't added to the team, which was rather surprising:

Reply:
Ah, interesting, so if any fail it backs out the whole set of changes. I suppose that's sensible of it.

How strange to have something upstream doing something that seems reasonable, heh